### PR TITLE
fixed issue #1

### PR DESCRIPTION
**Pull request by EphraimElvis**
Added new html, head and body tags

### CODE REVIEW

File: myproject.html [hunk 1]
```diff
@@ -1 +1,7 @@
-<h1>This is a project</h1>
+<html>
+  <head>
+    <title>Project</title>
+  </head>
```
Comment (friendly-developer) [inline myproject.html:4]: Add a meta tag

**Review by friendly-developer — CHANGES_REQUESTED** [review]
Please fix the above changes

File: myproject.html (hunk 1 above)
Comment (EphraimElvis) [reply myproject.html:4]: I will work on it

**Review by friendly-developer — APPROVED** [review]
Everything looks good

**Pull request merged**